x-pack/plugins/observability_solution/apm/server/routes/services/get_service_instance_container_metadata.ts

```diff
@@ -54,6 +55,7 @@ export const getServiceInstanceContainerMetadata = async ({
   const response = await infraMetricsClient.search({
     size: 1,
     track_total_hits: false,
+    fields: ['kubernetes*'],
```

Comment:
Question: Are we just interested in kubernetes containers? not docker?
I don't even know if that's an option I guess not, but just wondering

Reply:
It's ok, let's not change any fields if there were already like this